packages/next/src/build/index.ts

```diff
@@ -1306,6 +1310,38 @@ export default async function build(
         })
 
       async function turbopackBuild(): Promise<never> {
+        if (!process.env.TURBOPACK || !process.env.TURBOPACK_BUILD) {
+          throw new Error("next build doesn't support turbopack yet")
+        }
+        const bindings = await loadBindings(config?.experimental?.useWasmBinary)
+        const project = await bindings.turbo.createProject({
+          projectPath: dir,
+          rootPath: config.experimental.outputFileTracingRoot || dir,
+          nextConfig: config.nextConfig,
+          jsConfig: await getTurbopackJsConfig(dir, config),
+          watch: true,
+          env: process.env as Record<string, string>,
+          defineEnv: createDefineEnv({
+            isTurbopack: true,
+            allowedRevalidateHeaderKeys: undefined,
+            clientRouterFilters: undefined,
+            config,
+            dev: true,
+            distDir,
+            fetchCacheKeyPrefix: undefined,
+            hasRewrites,
+            middlewareMatchers: undefined,
+            previewModeId: undefined,
+          }),
+          // TODO: fix
+          // serverAddr: `127.0.0.1:${opts.port}`,
+          serverAddr: `127.0.0.1:3000`,
+        })
+
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const iter = project.entrypointsSubscribe()
```

Comment:
We should expose a non-subscribing method here

Reply:
Yeah this was just to get to the point where it can be called 👍